src/content/resources/language/evolution.md

```diff
@@ -42,6 +42,36 @@ on the Dart language GitHub repo.
 
 ## Changes in each release
 
+### Dart 3.8
+_Released 20 May 2025_
+| [Dart 3.8 announcement](https://medium.com/dartlang/...)
+
+The following language features have been added to Dart 3.8:
+
+*   [Null-aware elements][]: A null-aware element evaluates
+    an expression and if the result is not null, inserts the
+    value into the surrounding collection.
+
+*   [Doc imports][]: Dart now supports a
```

Comment:
Same as my comment about cross-compilation, I recommend removing this. It's not a change to the Dart language, but rather a new tooling feature in the SDK.

Reply:
Removed these.